README.md

```diff
@@ -38,8 +38,6 @@ timely manner. A secondary benefit is that it is easier to support a wide range
   You will need to add a `permit` line in `/etc/doas.conf` for it to work. This is not strictly
   necessary but since this is still experimental 
 * Replace the `common-auth` include in `/etc/pam.d/sudo` with `auth  required   pam_ssh_agent.so`
-* Configure `sudo` to not drop the `SSH_AUTH_SOCK` environment variable by
```

Comment:
I am pretty sure this is still needed. At a minimum, such a change should go in a separate PR